packages/core/components/checkbox/index.js

```diff
@@ -101,20 +122,20 @@ const Checkbox = ({
         required={required}
         aria-invalid={invalid}
       />
-      <StyledIcon
-        className="toggle--checked"
+      <Icon
```

Comment:
Do you think we should be applying `aria-hidden="true"` to the `<Icons>` in `<Checkbox>` and `<Radio`>

I was reading this and wanted your thoughts. https://css-tricks.com/accessible-svgs/#article-header-id-10

Reply:
Good catch. Yes, you're right.